src/ts/interfaces/ITelemetryData.ts

```diff
@@ -1,16 +1,39 @@
-interface ITelemetryEvent {
-  // The type of event we're sending, e.g. 'TYPERIGHTER_SUGGESTION_ACCEPTED' | 'TYPERIGHTER_MATCH_CREATED'
+export interface ITelemetryEvent {
+  /**
+   * The application sending the event
+   */
+  app?: string;
```

Comment:
These properties are optional here but mandatory in the telemetry service definition.